gtdynamics/utils/ChainInitializer.cpp

```diff
@@ -0,0 +1,101 @@
+/* ----------------------------------------------------------------------------
+ * GTDynamics Copyright 2020, Georgia Tech Research Corporation,
+ * Atlanta, Georgia 30332-0415
+ * All Rights Reserved
+ * See LICENSE for the license information
+ * -------------------------------------------------------------------------- */
+
+/**
+ * @file  ChainInitializer.h
+ * @brief Utility methods for initializing trajectory optimization solutions for chain graph.
+ * @author: Dan Barladeanu
+ */
+
+#include <gtdynamics/utils/ChainInitializer.h>
+
+namespace gtdynamics {
+
+/*Values Initializer::ZeroValues(const Robot& robot, const int t, double gaussian_noise,
```

Comment:
Kill commented out code. If you need to save some code, put it in a local branch

Reply:
done